### PR TITLE
DS3231: set & get aging offset register

### PR DESCRIPTION
**Pull request by dizcza**
I've extended the ds3231 module to set and get the aging offset register of an RTC.

### CODE REVIEW

**Comment by trombik** [comment]
your branch is outdated. would you rebase the branch with the `master` branch?

File: components/ds3231/ds3231.c [hunk 1]
```diff
@@ -421,3 +421,37 @@ esp_err_t ds3231_get_time(i2c_dev_t *dev, struct tm *time)
...
+     */
+    I2C_DEV_CHECK(dev, ds3231_set_flag(dev, DS3231_ADDR_CONTROL, DS3231_CTRL_TEMPCONV, DS3231_SET));
+
+    I2C_DEV_GIVE_MUTEX(dev);
+
+    return ESP_OK;
+}
+
+
+esp_err_t ds3231_get_aging_offset(i2c_dev_t *dev, int8_t *age) {
```
Comment (UncleRus) [inline components/ds3231/ds3231.c:445]: Curly brace must be on new line

File: components/ds3231/ds3231.c [hunk 2]
```diff
@@ -421,3 +421,37 @@ esp_err_t ds3231_get_time(i2c_dev_t *dev, struct tm *time)
 
     return ESP_OK;
 }
+
+
+esp_err_t ds3231_set_aging_offset(i2c_dev_t *dev, int8_t age) {
+    uint8_t age_u8 = (uint8_t) age;
```
Comment (UncleRus) [inline components/ds3231/ds3231.c:427]: `CHECK_ARG(dev)`

File: components/ds3231/ds3231.c [hunk 3]
```diff
@@ -421,3 +421,37 @@ esp_err_t ds3231_get_time(i2c_dev_t *dev, struct tm *time)
...
+    I2C_DEV_CHECK(dev, ds3231_set_flag(dev, DS3231_ADDR_CONTROL, DS3231_CTRL_TEMPCONV, DS3231_SET));
+
+    I2C_DEV_GIVE_MUTEX(dev);
+
+    return ESP_OK;
+}
+
+
+esp_err_t ds3231_get_aging_offset(i2c_dev_t *dev, int8_t *age) {
+    CHECK_ARG(age);
```
Comment (UncleRus) [inline components/ds3231/ds3231.c:446]: `CHECK_ARG(dev && age)`

**Comment by dizcza** [comment]
Done.

**Comment by UncleRus** [comment]
Thank you!

**Pull request merged**